### ISSUE TITLE
Help, train on voc

### ISSUE THREAD

**Issue by caihaunqai** [Andrew-Qibin/ssdlite-pytorch-mobilenext#5]
Hello, thanks for your excellent job. I use it to train voc datasets. After 40000 iterations, the results are as follows. It seems the miou only 7%? Can you help me, where I am wrong?
2021-09-15 03:23:16,957 SSD.trainer INFO: Total training time: 2 days, 9:11:49 (5.1477 s / it)
2021-09-15 03:23:16,981 SSD INFO: Start evaluating...
2021-09-15 03:23:17,036 SSD.inference INFO: Evaluating voc_2007_test dataset(4952 images):
2021-09-15 03:24:05,197 SSD.inference WARNING: Number of images that were gathered from multiple processes is not a contiguous set. Some images might be missing from the evaluation
2021-09-15 03:24:37,639 SSD.inference INFO: mAP: 0.0723
aeroplane       : 0.0652
bicycle         : 0.0396
bird            : 0.1066
boat            : 0.1150
bottle          : 0.0238
bus             : 0.0395
car             : 0.0527
cat             : 0.1141
chair           : 0.1031
cow             : 0.0233
diningtable     : 0.0628
dog             : 0.0737
horse           : 0.0834
motorbike       : 0.0736
person          : 0.0704
pottedplant     : 0.0180
sheep           : 0.1083
sofa            : 0.0568
train           : 0.1090
tvmonitor       : 0.1067

**Comment by caihaunqai** [comment]
The loss is here: 2021-09-15 03:23:15,876 SSD.trainer INFO: iter: 040000, lr: 0.00006, total_loss: 3.207 (4.156), cls_loss: 2.235 (2.905), reg_loss: 0.972 (1.250), time: 2.449 (5.148), eta: 0:00:00, mem: 19377M

**Comment by caihaunqai** [comment]
Done
